flyway-commandline/src/main/assembly/flyway.conf

```diff
@@ -21,6 +21,7 @@
 # * = driver must be downloaded and installed in /drivers manually
 # DB2*              : jdbc:db2://<host>:<port>/<database>
 # Derby             : jdbc:derby:<subsubprotocol:><databaseName><;attribute=value>
+# Greenplum*        : jdbc:pivotal:greenplum://<host>:<port>;DatabaseName=<database>;
 # H2                : jdbc:h2:<file>
```

Comment:
Feel free to include the JDBC driver by default in the Command-line distribution